test/tool/test-data/slice-data.json

```diff
@@ -0,0 +1,104 @@
+{
+  "inputsData": {
+    "float64Data": [
+      28.846251144212147,
+      97.9541470229301,
+      -68.15961736262523,
+      14.978987588557175,
+      90.23090084798065,
+      76.59095547712559,
+      -24.556316258204532,
+      79.58749302462488,
+      65.21375982234204,
+      57.43970862960305,
+      74.41775256222849,
+      -4.513182026141791,
+      0.5424534704526991,
+      80.44634130202977,
+      28.327650022124686,
+      74.0261938866893,
+      -74.54558964005362,
+      -27.30604081001799,
+      -70.4277433212161,
+      59.82631931755259,
+      -58.46095416849133,
+      79.80571087632629,
+      -9.857854207596304,
+      42.66519880465225
+    ]
+  },
+  "expectedData": {
+    "float321DDataDefault": [
+      0.5424534678459167,
+      80.44634246826172,
+      28.32765007019043,
+      74.02619171142578,
+      -74.54559326171875,
+      -27.306041717529297,
+      -70.42774200439453,
+      59.82632064819336,
+      -58.46095275878906,
+      79.80570983886719,
+      -9.857853889465332,
+      42.665199279785156
+    ],
+    "float322DDataDefault": [
+      28.32765007019043,
+      74.02619171142578,
+      -74.54559326171875,
+      -27.306041717529297,
+      -58.46095275878906,
+      79.80570983886719,
+      -9.857853889465332,
+      42.665199279785156
+    ],
+    "float323DDataDefault": [
```

Comment:
The name is tough to read. Is it a float323 with Data? Or a float3 with 23 dimensional data? :b How about a hyphen between to separate them (like phone numbers 503-749-2345)?

e.g. `float32-3DDataDefault`